docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -12,29 +12,58 @@ Office Add-ins are web applications that are displayed using iFrames when runnin
 
 - The computer's operating system.
-- Whether the add-in is running in Office on the web, Microsoft 365, or non-subscription Office 2013 or later.
+- Whether the add-in is running in Office on the web, Microsoft 365, or perpetual (also called "non-subscription" or "one-time purchase") Office 2013 or later.
+- Within the perpetual versions of Office, whether the add-in is running in the "consumer" or "commercial" (also called "volume-licensed" or "LTSC") variation.
 
 > [!IMPORTANT]
 > **Internet Explorer still used in Office Add-ins**
 >
-> Some combinations of platforms and Office versions, including one-time-purchase versions through Office 2019, still use the webview control that comes with Internet Explorer 11 to host add-ins, as explained in this article. We recommend (but don't require) that you continue to support these combinations, at least in a minimal way, by providing users of your add-in a graceful failure message when your add-in is launched in the Internet Explorer webview. Keep these additional points in mind:
+> Some combinations of platforms and Office versions, including perpetual versions through Office 2019, still use the webview control that comes with Internet Explorer 11 to host add-ins, as explained in this article. We recommend (but don't require) that you continue to support these combinations, at least in a minimal way, by providing users of your add-in a graceful failure message when your add-in is launched in the Internet Explorer webview. Keep these additional points in mind:
 >
 > - Office on the web no longer opens in Internet Explorer. Consequently, [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) no longer tests add-ins in Office on the web using Internet Explorer as the browser.
 > - AppSource still tests for combinations of platform and Office *desktop* versions that use Internet Explorer, however it only issues a warning when the add-in does not support Internet Explorer; the add-in is not rejected by AppSource.
 > - The [Script Lab tool](../overview/explore-with-script-lab.md) no longer supports Internet Explorer.
 >
 > For more information about supporting Internet Explorer and configuring a graceful failure message on your add-in, see [Support Internet Explorer 11](../develop/support-ie-11.md).
 
-The following table shows which browser is used for the various platforms and operating systems.
+The following sections specify which browser is used for the various platforms and operating systems.
+
+## Non-Windows Platforms
+
+For these platforms, the platform alone determines the browser that is used.
+
+|OS|Office version|Browser|
+|:-----|:-----|:-----|
+|any|Office on the web|The browser in which Office is opened.<br>(But note that Office on the web will not open in Internet Explorer.<br>Attempting to do so opens Office on the web in Edge.) |
+|Mac|any|Safari with WKWebView|
+|iOS|any|Safari with WKWebView|
+|Android|any|Chrome|
+
+## Perpetual Office on Windows
```

Comment:
```suggestion
## Perpetual versions of Office on Windows
```
Just want to avoid any branding confusing. "Perpetual Office" isn't a product; it's Office 2019 Perpetual, for example. Adding "versions of" for clarity on this.